src/vs/workbench/contrib/comments/browser/commentsController.ts

```diff
@@ -1162,9 +1196,9 @@ export class CommentController implements IEditorContribution {
 
 				this.displayCommentThread(info.owner, thread, pendingComment, pendingEdits);
 			});
-			info.pendingCommentThreads?.forEach(thread => {
-				this.commentService.createCommentThreadTemplate(thread.owner, thread.uri, Range.lift(thread.range));
-			});
+			for (const thread of info.pendingCommentThreads ?? []) {
+				this.resumePendingComment(this.editor!.getModel()!.uri, thread);
```

Comment:
Intentionally not awaited?

Reply:
For this call, it doesn't really mater either way. I'll make a PR addinging for consistency.